RecoLocalCalo/HcalRecProducers/python/HFPreReconstructor_cfi.py

```diff
@@ -0,0 +1,8 @@
+import FWCore.ParameterSet.Config as cms
+
+hfprereco = cms.EDProducer("HFPreReconstructor",
```

Comment:
please keep the file cfi file names match the module instance name defined in it.
``` git mv HFPreReconstructor_cfi.py hfprereco_cfi.py ```